internal/credentials.go

```diff
@@ -30,6 +30,7 @@ func (cr *Credentials) locations() []string {
 	return []string{
 		"/etc/zypp/credentials.d/SCCcredentials",
 		"/run/secrets/credentials.d/SCCcredentials",
+		"/run/secrets/SCCcredentials",
```

Comment:
Does this mean that `docker build` on a SLE machine (which uses `--mount=type=secret,id=SCCcredentials`) would still use the host's credentials?

Reply:
Yes, the possible paths will handled as priority because of the for loop in [configuration.go](https://github.com/SUSE/container-suseconnect/blob/master/internal/configuration.go#L55).

Reply:
I think the user mount should be given higher priority.